
<!-- table 5 saved from url=(0233)https://molpharm.aspetjournals.org/highwire/markup/49743/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed -->
<html lang="en" dir="ltr" version="HTML+RDFa+MathML 1.1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dc="http://purl.org/dc/terms/" xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns:og="http://ogp.me/ns#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:sioc="http://rdfs.org/sioc/ns#" xmlns:sioct="http://rdfs.org/sioc/types#" xmlns:skos="http://www.w3.org/2004/02/skos/core#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:mml="http://www.w3.org/1998/Math/MathML"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script type="text/javascript">
<!--//--><![CDATA[//><!--
setTimeout(function(){jQuery('.highwire-markup-wrapper[data-unique-id=9781f6a366527206702335f61aa70222]').width('985px');}, 20);
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
</head><body><div data-unique-id="9781f6a366527206702335f61aa70222" class="highwire-markup-wrapper highwire-get-markup" style="height:500px; width:1000px"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T3"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">TABLE 5</span> 
     
Koukouritaki SB, Poch MT, Cabacungan ET, McCarver DG, Hines RN. Discovery of novel flavin-containing monooxygenase 3 (FMO3) single nucleotide polymorphisms and functional analysis of upstream haplotype variants. Molecular pharmacology. 2005 Aug 1;68(2):383-92.
<p> all upstream SNPs. FMO3 sequence analysis revealed 7 upstream
variants, none of which have been reported in the literature, but 5 of which were reported in
build 124 of dbSNP (Table 5) (http://www.ncbi.nlm.nih.gov). 

	 <p id="p-17">Human <em>FMO3</em> SNP discovery—5′-flanking sequences</p>
            <p id="p-18">Coordinates are based on assigning the “A” of the ATG translation initiation codon +1. Accession number is that reported in dbSNP build 124.</p>
          <div class="sb-div caption-clear"></div></div><table id="table-3"><thead id="thead-3"><tr id="tr-33"><th colspan="1" rowspan="1" align="left" valign="bottom" id="th-9" class="table-left table-vbottom table-bborder"> Position </th><th colspan="1" rowspan="1" align="center" valign="bottom" id="th-10" class="table-center table-vbottom table-bborder"> SNP and Sequence Context </th><th colspan="1" rowspan="1" align="center" valign="bottom" id="th-11" class="table-center table-vbottom table-bborder"> Accession No. </th></tr></thead><tbody id="tbody-3"><tr id="tr-34"><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-135" class="table-center table-vbottom"> −2650 </td><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-136" class="table-center table-vbottom"> ATCAA[C&gt;G]GATTG </td><td colspan="1" rowspan="1" align="left" valign="bottom" id="td-137" class="table-left table-vbottom"> rs1736560 </td></tr><tr id="tr-35"><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-138" class="table-center table-vbottom"> −2589 </td><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-139" class="table-center table-vbottom"> AATCT[C&gt;T]TCATA </td><td colspan="1" rowspan="1" align="left" valign="bottom" id="td-140" class="table-left table-vbottom"></td></tr><tr id="tr-36"><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-141" class="table-center table-vbottom"> −2543 </td><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-142" class="table-center table-vbottom"> AGGCA[T&gt;A]GAAGG </td><td colspan="1" rowspan="1" align="left" valign="bottom" id="td-143" class="table-left table-vbottom"> rs12404218 </td></tr><tr id="tr-37"><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-144" class="table-center table-vbottom"> −2177 </td><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-145" class="table-center table-vbottom"> GAAAA[G&gt;C]AGAGA </td><td colspan="1" rowspan="1" align="left" valign="bottom" id="td-146" class="table-left table-vbottom"> rs3754491 </td></tr><tr id="tr-38"><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-147" class="table-center table-vbottom"> −2106 </td><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-148" class="table-center table-vbottom"> ATTTG[G&gt;A]AGGAC </td><td colspan="1" rowspan="1" align="left" valign="bottom" id="td-149" class="table-left table-vbottom"> rs16864006 </td></tr><tr id="tr-39"><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-150" class="table-center table-vbottom"> −2099 </td><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-151" class="table-center table-vbottom"> GGACT[A&gt;G]CTATG </td><td colspan="1" rowspan="1" align="left" valign="bottom" id="td-152" class="table-left table-vbottom"></td></tr><tr id="tr-40"><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-153" class="table-center table-vbottom table-bborder"> −1961 </td><td colspan="1" rowspan="1" align="center" valign="bottom" id="td-154" class="table-center table-vbottom table-bborder"> GAGTA[T&gt;C]AACCA </td><td colspan="1" rowspan="1" align="left" valign="bottom" id="td-155" class="table-left table-vbottom table-bborder"> rs16864007 </td></tr></tbody></table><div class="table-foot"></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></body></html>
		  
		  
		  
		  We used Taqman methods to type four specific variants, three in the FMO3 gene (rs2266780, rs2266782, and rs2066532) and one in the PYROXD2 gene (rs7072216). 
		   The PYROXD2 variant is located in the intronic region thus not captured by WES, but it was reported in a previous paper for significant association with TMA in urine
		   
		   <p>
		   We identified one rare allele within the FMO3 gene. This subject (#99) was heterozygous for a nonsense variant rs72549325 (chr1:g.171076936G > T; NM_001002294.2:c.442G > T; NP_001002294.1:p.Gly148Ter). We also identified subjects with common variants. Five subjects were heterozygous for rs2266782 (chr1:g.171076966G > A; NM_001002294.2:c.472G > A; NP_001002294.1:p.Glu158Lys), one subject was heterozygous for rs1736557 (chr1:g.171080080G > A; NM_001002294.2:c.769G > A; NP_001002294.1:p.Val257Met) and one subject was heterozygous for rs2266780 (chr1:g.171083242A > G; NM_001002294.2:c.923A > G; NP_001002294.1:p.Glu308Gly). Although common in the population, these three variants of the FMO3 gene have been associated with impairments in metabolism
		   
		   <p>
		   Shepard:
		   Some nonsynonymous variants, when present in cis configuration (e.g., p.Glu158Lys and p.Glu308Gly) result in moderately decreased enzyme activity [Phillips & Shephard 2020]. In the homozygous state (i.e., p.Glu158Lys and p.Glu308Gly present in cis configuration on both chromosomes), they may cause mild or transient primary trimethylaminuria, particularly in infants and young children [Zschocke & Mayatepek 2000], who have low expression of FMO3 [Koukouritaki et al 2002].

Although the rare variant p.Val187Ala alone does not affect enzyme activity, a combination of this variant in 
cis configuration with p.Glu158Lys severely affects enzyme activity and contributes to severe trimethylaminuria 
[Motika et al 2009].

<p>
Guo Y, Hwang LD, Li J, Eades J, Yu CW, Mansfield C, Burdick-Will A, Chang X, Chen Y, Duke FF, Zhang J. Genetic analysis of impaired trimethylamine metabolism using whole exome sequencing. BMC medical genetics. 2017 Dec;18(1):1-9.
<p>
    <div class="">
        
        <div id="mc" class="article objectonly"><div id="mc" class="content lit-style article"><div class="whole_rhythm"><!--main-content--><!--table-wrap mode="table" t7--><div class="whole_rhythm" id="Tab1"><h1 class="content-title">Table 1</h1><!--caption a7--><div class="caption"><p id="__p18">Characteristics of subjects with body odor complaints</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140328429529472" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">ID</th><th rowspan="1" colspan="1">Age (yr)</th><th rowspan="1" colspan="1">Sex</th><th rowspan="1" colspan="1">Race</th><th rowspan="1" colspan="1">TMAO: TMA<sup>a</sup>
</th><th rowspan="1" colspan="1">Fishy body odor</th><th rowspan="1" colspan="1">Fishy oral odor</th><th rowspan="1" colspan="1">Other, body odor</th><th rowspan="1" colspan="1">Common oral odors</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">52</td><td rowspan="1" colspan="1">78</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">C</td><td align="center" rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">Pungent, sulfurous<sup>b</sup>
</td></tr><tr><td rowspan="1" colspan="1">114</td><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">AA</td><td align="center" rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">Sulfurous/fecal<sup>b</sup>
</td></tr><tr><td rowspan="1" colspan="1">122</td><td rowspan="1" colspan="1">64</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">AA</td><td align="center" rowspan="1" colspan="1">0.47</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">Fecal, pungent<sup>b</sup>
</td></tr><tr><td rowspan="1" colspan="1">35</td><td rowspan="1" colspan="1">45</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">C</td><td align="center" rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">Mild sulfurous<sup>b</sup>
</td></tr><tr><td rowspan="1" colspan="1">99</td><td rowspan="1" colspan="1">59</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">AA</td><td align="center" rowspan="1" colspan="1">0.58</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">Mild metallic, smoky</td></tr><tr><td rowspan="1" colspan="1">64</td><td rowspan="1" colspan="1">51</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">C</td><td align="center" rowspan="1" colspan="1">0.61</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">Mild</td></tr><tr><td rowspan="1" colspan="1">62</td><td rowspan="1" colspan="1">51</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">AA</td><td align="center" rowspan="1" colspan="1">0.79</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">Sulfurous<sup>b</sup>
</td></tr><tr><td rowspan="1" colspan="1">113</td><td rowspan="1" colspan="1">47</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">C</td><td align="center" rowspan="1" colspan="1">0.79</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">Strong sulfurous<sup>b</sup>
</td></tr><tr><td rowspan="1" colspan="1">98</td><td rowspan="1" colspan="1">44</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">C</td><td align="center" rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">Musty<sup>c</sup>
</td><td rowspan="1" colspan="1">Unremarkable</td></tr><tr><td rowspan="1" colspan="1">56</td><td rowspan="1" colspan="1">70</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">C</td><td align="center" rowspan="1" colspan="1">0.87</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">None</td><td rowspan="1" colspan="1">Unremarkable</td></tr></tbody></table>
<em>ID</em> subject identifier number, <em>age</em> age at assessment in years (yr), <em>M</em> male, <em>F</em> female, <em>AA</em> American of African descent, <em>C</em> Caucasian American of European descent. Race/ethnicity was determined by self-report. <sup>a</sup>Table is sorted by TMAO:TMA ratio; Less than 0.90 is criterion for TMAU (i.e., reference &#x0003e; 0.90).<sup>b</sup>Oral odor secondary to plaque located on the posterior dorsal surface of the tongue. Plaque often contains odor-causing bacteria ([<a href="/pmc/articles/PMC5310055/#CR7" rid="CR7" class=" bibr" target="_blank" id="__tag_590105763">7</a>] and references therein). <sup>c</sup>Also described as &#x02018;damp&#x02019;</p></div></div></div></div></div>
<p>

<p>

Fisher’s test applied to calculate statistical significance revealed no
significant association between HSPA1A SNPs and TMAU phenotype. In
contrast, 7 SNPs in the HSPA8 gene were highly represented in the
patients, when compared to the healthy controls (Table 4). Of these SNPs,
the rs1136141c.-11C>T affects the 5’-UTR; and rs4935825, rs10892958,
rs3057456, rs201521469, and rs770829808 fall within non-coding exons,
spanning 100 nucleotides from the exon splice sites. The rs1064585
c.1455A>C; p.Ile485= is a synonymous variant. 
<p>
from Scimone C, Alibrandi S, Donato L, Esposito T, Sidoti A, D’Angelo R. VARIANTS OF THE MOLECULAR CHAPERONE HSPA8 AND HSPA1A GENES IN TRIMETHYLAMINURIA: A PILOT STUDY.
<p>
rs100848, 11557922, 201753050, 200771637, 1043618, 760554980, 2242667, 104620, 562047, 1219059, 1061581, 506770 - Tabl1, FMO variants
<p>
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6498825/
<p>
https://www.ncbi.nlm.nih.gov/books/NBK1103/table/trimethylaminuria.T.notable_fmo3_pathoge/?report=objectonly